apps/calendar/style/multi_day.css

```diff
@@ -39,6 +39,7 @@
   width: 7rem;
   top: 0;
   left: 0;
+  text-align: center;
```

Comment:
this should be on `.md__all-day` instead of focused only on the day view. otherwise the icon will be cropped on week view.